src/components/form-helpers/form-helpers.js

```diff
@@ -139,14 +179,14 @@ export const InputFieldItem = ({ className, fieldName, label, type, placeholder
   </FieldItem>
 );
 
-export const PasswordFieldItem = ({ className, fieldName, label, placeholder, helpText }) => {
+export const PasswordFieldItem = ({ className, fieldName, label, placeholder }) => {
   const [passVisible, setPassVisible] = useState(false);
   const type = passVisible ? 'text' : 'password';
   const autocomplete = passVisible ? 'off' : 'current-password';
   const buttonClasses = passVisible ? 'show-hide icon-hide ms-icon' : 'show-hide ms-icon icon-view';
 
   return (
-    <FieldItem className={concatClasses('field-item', className)} fieldName={fieldName}>
+    <PasswordWrapper className={concatClasses('field-item', className)} fieldName={fieldName}>
```

Comment:
Oh, take care of this. This PasswordWrapper does not apply for all password fields, only for ones who define password: _signup_ and _change password_, but not for _login_.

Reply:
![image](https://user-images.githubusercontent.com/1157864/55998104-4212ff80-5c93-11e9-8b82-096244b37668.png)

It does not look bad, but, in my opinion, as I talked with Santiago, it is not a good idea having this kind of validation on password validation fields because: imagine that the rules changed, the old password follow the old rules and the message could be confusing.

Reply:
I didn't notice this, thanks!

Reply:
Fixed this.